server/conn.go

```diff
@@ -394,7 +394,19 @@ func (cc *clientConn) writeInitialHandshake(ctx context.Context) error {
 }
 
 func (cc *clientConn) readPacket() ([]byte, error) {
-	return cc.pkt.readPacket()
+	// Default MaxAllowedPacket is 16MB.
```

Comment:
It should be `64MB` according to:

* https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_allowed_packet
* https://github.com/pingcap/tidb/blob/69f664ae051c21e0edc48fcc5556f19fe4465a9a/sessionctx/variable/sysvar.go#L845

Although we have:

> The default [max_allowed_packet](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_allowed_packet) value built in to the client library is 1GB, but individual client programs might override this. For example, [mysql](https://dev.mysql.com/doc/refman/8.0/en/mysql.html) and [mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) have defaults of 16MB and 24MB, respectively.

The default of `16MB` is set by the `mysql` client.

Reply:
Also, please define a constant named like `DefMaxAllowedPacket` here: https://github.com/pingcap/tidb/blob/69f664ae051c21e0edc48fcc5556f19fe4465a9a/sessionctx/variable/tidb_vars.go/#L673

And use this constant for the default values.